convertPostfitShapesForPlotIt.py

```diff
@@ -48,7 +48,8 @@ def remove_errors(hist):
 
 processs = set()
 
-for proc in file.Get('%s_prefit' % channels[0]).GetListOfKeys():
+#for proc in file.Get('%s_prefit' % channels[0]).GetListOfKeys():
+for proc in file.Get('%sb2j3_prefit' % channels[0][:-4]).GetListOfKeys():
```

Comment:
why is that?

Reply:
Also for QCD. QCD process is only included in b2j3 so I specified b2j3 in the line